### PR TITLE
Update docs for 4.1

### PR DESCRIPTION
**Pull request by mtnbikenc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,25 @@
...
-tracks our current work **in development** and should be compatible
-with the
-[Origin master branch](https://github.com/openshift/origin/tree/master)
-(code in development).
+## Previous OpenShift Ansible 3.x releases
+For 3.x releases of OpenShift Ansible please reference the release branch for
+specific versions.  The last 3.x release is 
+[3.11 release](https://github.com/openshift/openshift-ansible/tree/release-3.11).
 
+## OpenShift 4
```
Comment (DanyC97) [inline README.md]: for consistency (with the 3.x title) and future release, maybe you want to reword it like `OpenShift 4.x releases`

File: README.md [hunk 2]
```diff
@@ -1,65 +1,25 @@
...
+Installation of OpenShift beginning with 4.0 uses a command-line installation
+wizard instead of Ansible playbooks.  Learn more about the OpenShift Installer
```
Comment (DanyC97) [inline README.md]: nit: maybe reword it like

> Installation of OpenShift 4.x uses a command-line wizard instead of Ansible playbooks

File: README.md [hunk 3]
```diff
@@ -1,65 +1,25 @@
...
+in this [overview](https://github.com/openshift/installer/blob/master/docs/user/overview.md#installer-overview).
 
+For OpenShift 4.0, this repo only provides playbooks necessary for scaling up an
+existing 4.0 cluster with RHEL hosts.
```
Comment (DanyC97) [inline README.md]: is this the case for 4.0 only while on 4.1+ the scope will be extended ?

regarding _RHEL hosts_ are are pinned to a particular version - ie >= 8.0 ?

Comment (mtnbikenc) [reply README.md]: Thanks for the input.  In this paragraph, I was attempting to provide a contrast between 3.x and 4.0 in that they are very different, but not get bogged down in a specific feature inclusion/exclusion list.  Something like, "Don't come looking for installation playbooks, all you'll find right now is a scaleup playbook."  I was also trying to avoid '4.x'.  I've pushed a small update.

File: README.md [hunk 4]
```diff
@@ -1,171 +1,42 @@
...
-
-```sh
-openshift-install create ignition-configs
-```
+<updated inventory requirements here>
 
-## Run playbook
-playbooks/deploy_cluster_40.yml
+## Run scaleup playbook
+ansible-playbook playbooks/openshift_node/scaleup.yml
```
Comment (patrickdillon) [inline README.md]: Code block should be surrounded with ticks

File: README.md (hunk 3 above)
Comment (DanyC97) [reply README.md]: fair point, thx for the info

**Comment by mtnbikenc** [comment]
/retest
```
Failing tests:

[Feature:DeploymentConfig] deploymentconfigs when run iteratively [Conformance] should only deploy the last deployment [Suite:openshift/conformance/parallel/minimal]
[k8s.io] EquivalenceCache [Serial] validates pod affinity works properly when new replica pod is scheduled [Suite:openshift/conformance/serial] [Suite:k8s]
[k8s.io] EquivalenceCache [Serial] validates pod anti-affinity works properly when new replica pod is scheduled [Suite:openshift/conformance/serial] [Suite:k8s]
[k8s.io] [sig-node] kubelet [k8s.io] [sig-node] Clean up pods on node kubelet should be able to delete 10 pods per node in 1m0s. [Suite:openshift/conformance/serial] [Suite:k8s] [Serial]
[sig-api-machinery] Aggregator Should be able to support the 1.7 Sample API Server using the current Aggregator [Suite:openshift/conformance/serial] [Suite:k8s] [Serial]
[sig-api-machinery] Namespaces [Serial] should always delete fast (ALL of 100 namespaces in 150 seconds) [Feature:ComprehensiveNamespaceDraining] [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-api-machinery] Namespaces [Serial] should delete fast enough (90 percent of 100 namespaces in 150 seconds) [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-api-machinery] Namespaces [Serial] should ensure that all pods are removed when a namespace is deleted [Conformance] [Suite:openshift/conformance/serial/minimal] [Suite:k8s]
[sig-apps] Daemon set [Serial] should not update pod when spec was updated and update strategy is OnDelete [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-apps] Daemon set [Serial] should run and stop complex daemon [Conformance] [Suite:openshift/conformance/serial/minimal] [Suite:k8s]
[sig-apps] Daemon set [Serial] should update pod when spec was updated and update strategy is RollingUpdate [Conformance] [Suite:openshift/conformance/serial/minimal] [Suite:k8s]
[sig-cli] Kubectl client [k8s.io] Kubectl taint [Serial] should update the taint on a node [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-scheduling] PodPriorityResolution [Serial] validates critical system priorities are created and resolved [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-scheduling] SchedulerPredicates [Serial] validates resource limits of pods that are allowed to run  [Conformance] [Suite:openshift/conformance/serial/minimal] [Suite:k8s]
[sig-scheduling] SchedulerPredicates [Serial] validates that NodeAffinity is respected if not matching [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-scheduling] SchedulerPredicates [Serial] validates that NodeSelector is respected if matching  [Conformance] [Suite:openshift/conformance/serial/minimal] [Suite:k8s]
[sig-scheduling] SchedulerPredicates [Serial] validates that NodeSelector is respected if not matching  [Conformance] [Suite:openshift/conformance/serial/minimal] [Suite:k8s]
[sig-scheduling] SchedulerPredicates [Serial] validates that required NodeAffinity setting is respected if matching [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-scheduling] SchedulerPredicates [Serial] validates that taints-tolerations is respected if matching [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-scheduling] SchedulerPredicates [Serial] validates that taints-tolerations is respected if not matching [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-scheduling] SchedulerPreemption [Serial] validates basic preemption works [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-scheduling] SchedulerPreemption [Serial] validates lower priority pod preemption by critical pod [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-scheduling] SchedulerPreemption [Serial] validates pod anti-affinity works in preemption [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-storage] In-tree Volumes [Driver: nfs] [Testpattern: Dynamic PV (default fs)] subPath should be able to unmount after the subpath directory is deleted [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-storage] In-tree Volumes [Driver: nfs] [Testpattern: Inline-volume (default fs)] subPath should be able to unmount after the subpath directory is deleted [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-storage] In-tree Volumes [Driver: nfs] [Testpattern: Pre-provisioned PV (default fs)] subPath should be able to unmount after the subpath directory is deleted [Suite:openshift/conformance/parallel] [Suite:k8s]
[sig-storage] [Serial] CSI Volumes CSI plugin test using CSI driver: gcePD [Serial] should provision storage [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-storage] [Serial] Volume metrics PVController should create unbound pvc count metrics for pvc controller after creating pvc only [Suite:openshift/conformance/serial] [Suite:k8s]
[sig-storage] [Serial] Volume metrics should create metrics for total number of volumes in A/D Controller [Suite:openshift/conformance/serial] [Suite:k8s]
```

**Comment by sdodson** [comment]
/override ci/prow/e2e-aws-scaleup-rhel7
docs only
